tests/idl/rapidjson_generator/Main.cpp

```diff
@@ -0,0 +1,7 @@
+#include "gtest/gtest.h"
```

Comment:
GoogleTest is only available conditionally.  We will have to discuss if we want to change that.

Reply:
I don't really know that we need this test to run all the time, since we're not interested in testing rapidJSON functionality so much as testing our interaction with it, so conditionally running it seems OK? I'm guessing there's something I can add to the MPC file to make it only get built / run when gtest is available...